packages/web-react/src/components/Toast/stories/UncontrolledToast.stories.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import { Markdown } from '@storybook/blocks';
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { ToastColorType, UncontrolledToastProps } from '../../../types';
+import { Button } from '../../Button';
+import ReadMe from '../README.md';
+import { useUncontrolledToast } from '../useUncontrolledToast';
+import { ToastProvider, UncontrolledToast } from '..';
+
+interface UncontrolledToastPlaygroundProps extends UncontrolledToastProps {
+  autoClose: boolean;
+  color: ToastColorType;
+  iconName: string;
+}
+
+const meta: Meta<UncontrolledToastPlaygroundProps> = {
+  title: 'Components/Toast',
+  component: UncontrolledToast,
+  parameters: {
+    docs: {
+      page: () => <Markdown>{ReadMe}</Markdown>,
+    },
+  },
+  argTypes: {
+    children: {
+      control: 'text',
+    },
+    alignmentX: {
+      control: 'select',
+      options: ['left', 'center', 'right'],
+      table: {
+        defaultValue: { summary: 'center' },
+      },
+    },
+    alignmentY: {
+      control: 'select',
+      options: ['top', 'bottom'],
+      table: {
+        defaultValue: { summary: 'bottom' },
+      },
+    },
+    closeLabel: {
+      control: 'text',
+    },
+    hasIcon: {
+      control: 'boolean',
+    },
+    isDismissible: {
+      control: 'boolean',
+    },
+    autoClose: {
+      control: 'boolean',
+    },
+    color: {
+      control: 'select',
+      options: ['inverted', 'danger', 'warning', 'success', 'info'],
```

Comment:
```suggestion
      options: ['inverted', ...Object.values(EmotionColors)],
```
can we do this? Also in the toastbar story.

Reply:
we can do this! thanks